.editorconfig

```diff
@@ -0,0 +1,19 @@
+# Top-most EditorConfig file
```

Comment:
@Carreau @gnestor thoughts on adding this file? It seems reasonable to me, but I know people sometimes take issue with ever more config files in the repo root directory.

Reply:
I'm ok adding it, but I don't know of a lot of people using that. It may likely just rot as the tox.ini file which is never used and still test on Python 27 as shown further in this PR.